librosa/util/_nnls.py

```diff
@@ -25,13 +26,13 @@ def _nnls_obj(x, shape, A, B):
     x = x.reshape(shape)
```

Comment:
I think you need order=F here to make sure that flatten/unflatten is done consistently

Reply:
Yeah, I changed `order=F` all over and the original `test_util.py` tests passed, but the same issue persisted with my test.